lib/__tests__/conversion.test.ts

```diff
@@ -54,3 +54,73 @@ describe("convertedObjectIDInSearch", () => {
     );
   });
 });
+describe("convertedObjectIDs", () => {
+  it("should throw if no params are sent", () => {
+    expect(() => {
+      AlgoliaInsights.init(credentials);
+      (AlgoliaInsights as any).convertedObjectIDs();
+    }).toThrowError(
+      "No params were sent to convertedObjectIDs function, please provide `objectIDs` to be reported"
+    );
+  });
+
+  it("should throw if no objectIDs has been passed", () => {
+    (AlgoliaInsights as any).sendEvent = jest.fn();
+    AlgoliaInsights.init(credentials);
+
+    expect(() => {
+      (AlgoliaInsights as any).convertedObjectIDs({ queryID: "test" });
```

Comment:
The `queryID` should not be present if I understood correctly, since we are not inside the context of a search. It should be caught by the TypeScript though but since we cast the object it does not see the error.

Reply:
removed queryID, you're right it has nothing to do there!